inttest/common/util.go

```diff
@@ -170,6 +170,18 @@ func WaitForDeployment(ctx context.Context, kc *kubernetes.Clientset, name, name
 		})
 }
 
+// WaitForStatefulSet waits for the StatefulSet with the given name to become
+// available as long as the given context isn't canceled.
+func WaitForStatefulSet(ctx context.Context, kc *kubernetes.Clientset, name, namespace string) error {
+	return watch.StatefulSet(kc.AppsV1().StatefulSets(namespace)).
+		WithObjectName(name).
+		WithErrorCallback(RetryWatchErrors(logrus.Infof)).
+		Until(ctx, func(ss *appsv1.StatefulSet) (bool, error) {
```

Comment:
(Germans always have to choke hard when reading `ss`, even in lower case. I think I have to come to terms with that.)

Reply:
Sorry that didn't even cross my mind. My first though of `ss`is the command for "socket statistics" I'll rename it to `s`.

Reply:
No worries. The abbreviation is the first thing that comes to mind and completely legitimate in this context. Thanks for changing anyways!